### PR TITLE
landing screen

### PR DESCRIPTION
**Pull request by manukyan-edgar**
[As a user I should be able to see the landing screen](https://trello.com/c/2VXuWcUh/4-as-a-user-i-should-be-able-to-see-the-landing-screen)

### CODE REVIEW

**Review by ManukyanQristine — APPROVED** [review]

File: app/src/main/java/com/simply/birthdayapp/presentation/navigation/AuthNavigation.kt [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    val nestedNavController = rememberNavController()
+    NavHost(
+        navController = nestedNavController,
+        startDestination = AuthScreen.LandingScreen.route,
+        route = "root",
+    ) {
+        composable(AuthScreen.LandingScreen.route) {
+            LandingScreen(
+                onSignInClick = { nestedNavController.navigate(AuthScreen.SignInScreen.route) },
+                onRegisterClick = { nestedNavController.navigate(AuthScreen.RegisterScreen.route) })
```
Comment (Hayk985) [inline app/src/main/java/com/simply/birthdayapp/presentation/navigation/AuthNavigation.kt]: Move the ending bracket to the next line

File: app/src/main/java/com/simply/birthdayapp/presentation/ui/screens/auth/AuthMainScreen.kt [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package com.simply.birthdayapp.presentation.ui.screens.auth
+
+import androidx.compose.runtime.Composable
+import com.simply.birthdayapp.presentation.navigation.AuthNavigation
+
+
+@Composable
+fun AuthMainScreen() {
+    AuthNavigation()
```
Comment (Hayk985) [inline app/src/main/java/com/simply/birthdayapp/presentation/ui/screens/auth/AuthMainScreen.kt:9]: Why do we need to add a composable which is just a holder for another composable? Do we plan to add some code or logic in AuthMainScreen for later? If not, no need to have another composable AuthNavigation. We can write the code directly in AuthMainScreen.

File: app/src/main/java/com/simply/birthdayapp/presentation/ui/components/LandingButton.kt
Comment (Hayk985) [inline app/src/main/java/com/simply/birthdayapp/presentation/ui/components/LandingButton.kt:1]: Add default values to arguments and create a preview for this composable.

File: app/src/main/java/com/simply/birthdayapp/presentation/ui/components/LandingButton.kt [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    onClick: () -> Unit = {}
+) {
+    Button(
+        modifier = Modifier
+            .padding(3.dp)
+            .height(51.dp)
+            .fillMaxWidth(),
+        shape = shape,
+        onClick = {
+            onClick()
```
Comment (Hayk985) [inline app/src/main/java/com/simply/birthdayapp/presentation/ui/components/LandingButton.kt]: No need to have lambda closures here

File: app/src/main/java/com/simply/birthdayapp/presentation/ui/screens/auth/landing/LandingScreen.kt [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .background(color = BackgroundColor)
+            .padding(horizontal = 54.dp),
+        verticalArrangement = Arrangement.Center,
+        horizontalAlignment = Alignment.CenterHorizontally,
+    ) {
+        Image(
+            painterResource(R.drawable.logo),
```
Comment (Hayk985) [inline app/src/main/java/com/simply/birthdayapp/presentation/ui/screens/auth/landing/LandingScreen.kt]: Add a named argument for painterResource too

File: app/src/main/java/com/simply/birthdayapp/presentation/ui/screens/auth/landing/LandingScreen.kt [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+        LandingButton(
+            shape = RoundedCornerShape(
+                topStart = 42.dp,
+                topEnd = 26.dp,
+                bottomEnd = 26.dp
+            ),
+            buttonTitle = stringResource(id = R.string.sign_in),
+            backgroundColor = Primary1,
+            textColor = Primary2,
+            onClick = { onSignInClick() })
```
Comment (Hayk985) [inline app/src/main/java/com/simply/birthdayapp/presentation/ui/screens/auth/landing/LandingScreen.kt:51]: No need to have lambda closures here too and move the ending bracket to the next line. 
Do the same for register button.

File: app/src/main/java/com/simply/birthdayapp/presentation/ui/screens/auth/register/RegisterScreen.kt [hunk 6]
```diff
@@ -0,0 +1,35 @@
+package com.simply.birthdayapp.presentation.ui.screens.auth.register
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+
+import androidx.compose.material3.Text
```
Comment (Hayk985) [inline app/src/main/java/com/simply/birthdayapp/presentation/ui/screens/auth/register/RegisterScreen.kt:8]: Format the code. We should not have blank lines here.

File: app/src/main/java/com/simply/birthdayapp/presentation/ui/screens/auth/register/RegisterScreen.kt [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+
+@Composable
+fun RegisterScreen() {
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .background(color = BackgroundColor),
+        verticalArrangement = Arrangement.Center,
+        horizontalAlignment = Alignment.CenterHorizontally,
+        ) {
```
Comment (Hayk985) [inline app/src/main/java/com/simply/birthdayapp/presentation/ui/screens/auth/register/RegisterScreen.kt:26]: Format the code here too.

File: app/src/main/java/com/simply/birthdayapp/presentation/ui/screens/auth/signIn/SingnInScreen.kt [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+
+@Composable
+fun SignInScreen() {
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .background(color = BackgroundColor),
+        verticalArrangement = Arrangement.Center,
+        horizontalAlignment = Alignment.CenterHorizontally,
+        ) {
```
Comment (Hayk985) [inline app/src/main/java/com/simply/birthdayapp/presentation/ui/screens/auth/signIn/SingnInScreen.kt]: Format the code here too.

File: app/src/main/java/com/simply/birthdayapp/presentation/navigation/AuthNavigation.kt (hunk 1 above)
Comment (manukyan-edgar) [reply app/src/main/java/com/simply/birthdayapp/presentation/navigation/AuthNavigation.kt]: ok

File: app/src/main/java/com/simply/birthdayapp/presentation/ui/components/LandingButton.kt
Comment (manukyan-edgar) [reply app/src/main/java/com/simply/birthdayapp/presentation/ui/components/LandingButton.kt:1]: ok

File: app/src/main/java/com/simply/birthdayapp/presentation/ui/components/LandingButton.kt (hunk 3 above)
Comment (manukyan-edgar) [reply app/src/main/java/com/simply/birthdayapp/presentation/ui/components/LandingButton.kt]: ok

File: app/src/main/java/com/simply/birthdayapp/presentation/ui/screens/auth/AuthMainScreen.kt (hunk 2 above)
Comment (manukyan-edgar) [reply app/src/main/java/com/simply/birthdayapp/presentation/ui/screens/auth/AuthMainScreen.kt:9]: We have create structure with navigation package. it is already approved and i just keep the  existing style

File: app/src/main/java/com/simply/birthdayapp/presentation/ui/screens/auth/landing/LandingScreen.kt (hunk 4 above)
Comment (manukyan-edgar) [reply app/src/main/java/com/simply/birthdayapp/presentation/ui/screens/auth/landing/LandingScreen.kt]: ok

File: app/src/main/java/com/simply/birthdayapp/presentation/ui/screens/auth/landing/LandingScreen.kt (hunk 5 above)
Comment (manukyan-edgar) [reply app/src/main/java/com/simply/birthdayapp/presentation/ui/screens/auth/landing/LandingScreen.kt:51]: ok

File: app/src/main/java/com/simply/birthdayapp/presentation/ui/screens/auth/register/RegisterScreen.kt (hunk 6 above)
Comment (manukyan-edgar) [reply app/src/main/java/com/simply/birthdayapp/presentation/ui/screens/auth/register/RegisterScreen.kt:8]: ok

File: app/src/main/java/com/simply/birthdayapp/presentation/ui/screens/auth/register/RegisterScreen.kt (hunk 7 above)
Comment (manukyan-edgar) [reply app/src/main/java/com/simply/birthdayapp/presentation/ui/screens/auth/register/RegisterScreen.kt:26]: ok

File: app/src/main/java/com/simply/birthdayapp/presentation/ui/screens/auth/signIn/SingnInScreen.kt (hunk 8 above)
Comment (manukyan-edgar) [reply app/src/main/java/com/simply/birthdayapp/presentation/ui/screens/auth/signIn/SingnInScreen.kt]: ok

Comment (manukyan-edgar) [reply app/src/main/java/com/simply/birthdayapp/presentation/ui/screens/auth/signIn/SingnInScreen.kt]: Already formated  with Option+Command+L

**Pull request merged**